etest_test/profile_test.py

```diff
@@ -0,0 +1,153 @@
+"""Tests for etest.profile."""
+
+import logging
+from typing import Any, Dict
+
+import pytest
+from hypothesis import assume, given
+from hypothesis import strategies as st
+from pytest_mock import MockerFixture
+
+import etest.profile as sut
+
+logger = logging.getLogger(__name__)
+
+
+_valid_architectures = ["amd64", "x86", "armv5", "armv6", "armv7", "arm64", "ppc64"]
+_valid_systemd_architectures = ["amd64", "x86", "arm64", "ppc64"]
+_libc_value = ["glibc", "musl", "uclibc"]
```

Comment:
These should be in the source and not in the test.  That's why I suggest making an Enum for each of these.  The code then will use the same values as the tests and there is no worry about divergence.  This makes all checks against this correct by definition as well which is nice.